server/services/twilio.js

```diff
@@ -18,7 +18,7 @@ const createToken = (identity) => {
   accessToken.identity = identity;
 
   const grant = new VideoGrant();
-  grant.room = 'Room 1';
+  grant.room = identity.room;
```

Comment:
this is an oversight of mine but you'll need to make some adjustments to createToken to accept a room argument and set it here.